### PR TITLE
New ways to connect

### PR DESCRIPTION
**Pull request by akshay196**
Added some new ways to connect to PythonPune:

- Add LinkedIn page URL in footer
- Add Telegram group in "join us" section.

### CODE REVIEW

**Comment by akshay196** [comment]
See this PR site preview at https://deploy-preview-154--pythonpune.netlify.app/

**Review by bhavin192 — APPROVED** [review]

**Comment by akshay196** [comment]
@vishalvvr Updated to new LinkedIn URL. PTAL

File: config.yaml [hunk 1]
```diff
@@ -114,6 +114,11 @@ params:
     subtitle: Easy to connect us with these open platforms
     row:
     - list:
+      - icon: fa-telegram
+        title: Telegram
+        description: >-
+          Join **PythonPune** telegram group to chat with us.
```
Comment (bhavin192) [inline config.yaml:120]:
```suggestion
          Join **PythonPune** Telegram group to chat with us (linked with the IRC channel).
```
How about this?

**Review by bhavin192 — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 1 above)
Comment (akshay196) [reply config.yaml:120]: Looks good. I will make the changes as suggested. :)

Comment (akshay196) [reply config.yaml]: Updated. Thanks for suggestion.

**Review by bhavin192 — APPROVED** [review]

**Review by vishalvvr — APPROVED** [review]

**Pull request merged**